shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -291,6 +292,16 @@ message DeleteScopeStatus {
     Status status = 1;
 }
 
+message DeleteScopeRecursiveStatus {
+    enum Status {
+        SUCCESS = 0;
+        FAILURE = 1;
+        SCOPE_NOT_FOUND = 2;
+        SCOPE_NOT_EMPTY = 3;
```

Comment:
Why would recursive deleteScope fail because SCOPE_NOT_EMPTY. We should never return that to the user.